applications/NXem.nxdl.xml

```diff
@@ -0,0 +1,1693 @@
+                </group>
+                <group name="scan_controller" type="NXscanbox_em" optional="true">
+                    <group name="fabrication" type="NXfabrication" recommended="true">
+                        <field name="vendor" type="NX_CHAR"/>
+                        <field name="model" type="NX_CHAR"/>
+                        <field name="serial_number" type="NX_CHAR" recommended="true"/>
+                    </group>
+                </group>
+                <group name="stage" type="NXmanipulator" minOccurs="0" maxOccurs="unbounded">
+                    <field name="design" type="NX_CHAR" recommended="true"/>
+                    <!--add enumeration values from old NXstage_lab-->
+                    <group name="fabrication" type="NXfabrication" recommended="true">
+                        <field name="vendor" type="NX_CHAR"/>
+                        <field name="model" type="NX_CHAR"/>
+                        <field name="serial_number" type="NX_CHAR" recommended="true"/>
+                    </group>
+                </group>
+                <group name="nanoprobe" type="NXmanipulator" optional="true">
+                    <group name="fabrication" type="NXfabrication" recommended="true">
+                        <field name="vendor" type="NX_CHAR"/>
+                        <field name="model" type="NX_CHAR"/>
+                        <field name="serial_number" type="NX_CHAR" recommended="true"/>
+                    </group>
+                </group>
+                <group type="NXpump" minOccurs="0" maxOccurs="unbounded">
+                    <doc>
+                        Instances should use pump as a name prefix.
+                    </doc>
+                    <field name="design" type="NX_CHAR"/>
+                </group>
+                <group type="NXsensor" minOccurs="0" maxOccurs="unbounded">
+                    <doc>
+                        Instances should use sensor as a name prefix.
+                    </doc>
+                </group>
+                <group type="NXactuator" minOccurs="0" maxOccurs="unbounded">
+                    <doc>
+                        Instances should use actuator as a name prefix.
+                    </doc>
+                </group>
+            </group>
+            <group name="events" type="NXobject" minOccurs="0" maxOccurs="1">
+                <!--an instance must not have an NXevent_data_em_set but if it has one it must not be more than 1 !-->
+                <doc>
+                    This group should be used to store all event-related (meta)data,
+                    which are typically measured datasets like images and spectra.
+                    To avoid that static instrument-related metadata need to be stored
+                    repetitively the NXem application definitions splits the storage of the
+                    dynamic (meta)data that typically change for each image and spectrum
+                    from the static one.
```

Comment:
Alternative:
Event-related (meta)data,  typically measured datasets like images and spectra. To avoid repetitively storing static instrument-related metadata,  the dynamic (meta)data that typically changes for each image and spectrum is split from the static (meta)data.

Reply:
Suggestion accepted in the new draft